src/main/scala/org/phenoscape/kb/Main.scala

```diff
@@ -1009,10 +1008,10 @@ object Main extends HttpApp with App {
               } ~
               pathPrefix("gene") {
                 path("search") {
-                  parameters('text, 'taxon.as[IRI].?) { (text, taxonOpt) => //FIXME add limit option?
+                  parameters('text, 'taxon.as[IRI].?, 'limit.as[Int].?(100)) { (text, taxonOpt, limit) =>
```

Comment:
Swagger yaml file would need to be updated with this parameter.

Reply:
Fixed.